data/schema.graphql

```diff
@@ -194,32 +194,32 @@ type ArtistCarousel {
 
 type ArtistCounts {
   artworks(
-    # Returns a `String` when format is specified. e.g.`"0,0.0000"`
+    # Returns a `String` when format is specified. e.g.`'0,0.0000''`
```

Comment:
Been converting all of the source code to use [prettier](https://github.com/prettier/prettier/), in doing so we switched from `'` to `"` - think I missed a spot and it got caught as I was working.